R/matrix_form.R

```diff
@@ -895,16 +895,18 @@ mf_has_rlabels <- function(mf) ncol(mf$strings) > mf_ncol(mf)
 #' cat(toString(mform))
 #'
 #' # Advanced test case with label rows
-#' library(dplyr)
-#' iris_output <- iris %>%
-#'   group_by(Species) %>%
-#'   summarize("all obs" = round(mean(Petal.Length), 2)) %>%
-#'   mutate("DataRow_label" = "Mean")
-#' mf <- basic_matrix_form(iris_output,
-#'   indent_rownames = TRUE,
-#'   split_labels = "Species", data_labels = "DataRow_label"
-#' )
-#' cat(toString(mf))
+#' if (require("dplyr")) {
```

Comment:
You can do multiple `@examples` section. Eventually this will be collapsed into a single one.

Here, I suggest `@examplesIf require("dplyr")`. That's the standard for conditional examples